cluster/manifests/skipper/deployment.yaml

```diff
@@ -1,43 +1,69 @@
 {{ $internal_version := "v0.18.42-691" }}
 {{ $version := index (split $internal_version "-") 0 }}
 
+{{ template "skipper-ingress" dict
+  "name" "skipper-ingress"
+  "internal_version" $internal_version
+
+  "Cluster" .Cluster
+  "Values" .Values
+}}
+
+{{ if .Cluster.ConfigItems.skipper_ingress_canary_internal_version }}
+{{ template "skipper-ingress" dict
+  "name" "skipper-ingress-canary"
+  "internal_version" .Cluster.ConfigItems.skipper_ingress_canary_internal_version
```

Comment:
config-item skipper_ingress_canary_internal_version TBD. I would really like to have a fixed version.
I would suggest to have only a `skipper_ingress_canary_enable: "true"` cfg item to allow enable and delete, but a fixed version until we really need a dynamic settable config item version.
One safety net is that we need to roll through all channels and this would enable us to switch bypassing all channels.